contents/handbook/people/team-structure/app-small-teams.md

```diff
@@ -0,0 +1,41 @@
+---
+title: App Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: false
+---
+
+App Small Teams are particular types of Small Teams that own an area of the product.
+
+## What does owning an area of the product mean?
+
+The app small team is responsible for everything related to that area of the product including:
+
+1. Usage
+2. Quality
+3. Revenue
+
+## What actions should the app small teams be doing for their area?
+
+- Each quarter:
+  1. Create good quarterly OKRs
+  2. Create an ambitious vision for their area
+- During the quarter:
+  1. Maintain a prioritized roadmap to help them achieve their objectives
```

Comment:
What problem does having a roadmap for each team solve and who will be it's main consumers?

(I wrote in-depth about why I'm not a fan of how roadmap-driven development usually falls out here: https://github.com/PostHog/posthog/issues/11749#issuecomment-1253302481)

Reply:
Does roadmap here really mean backlog? But as Karl says "roadmap" "backlog" and similar are overloaded and can reduce clarity

If we rewrote as `frequently prioritise upcoming work to remain focussed on their objectives`... does that help guide what we do without restricting how we do it?

Reply:
What Annika and Paul are saying with "frequently prioritise upcoming work to remain focussed on their objectives" is more what I'm thinking. It's not about having a strict list of stuff they must deliver, but more a space for them to keep track of potential ideas for the future sprints.

Reply:
I'll update this now

Reply:
What do you think about there being a more formal requirement that this is saved as a stack ranked list that is shared internally? We don't need to be prescriptive of the format or location but I know it would help me and likely others if there was a linked list written down somewhere?

Reply:
So the line would be "Frequently prioritize upcoming work to help them achieve their objectives (the format does not matter but the link to this should be shared)"

Reply:
That's fine by me... 

We need to keep each other honest so it doesn't become too large a list or too process-y to keep it up-to-date

("the planning is essential but the plans are worthless" - Dwight D. Eisenhower)

Reply:
I'm not sure what line should end up in this doc, but at least in Team Product Analytics, we're dumping every relevant issue that comes our way onto [the project's board here](https://github.com/orgs/PostHog/projects/59), and at least each week coming back to see what urgent bugs are there to distract us from our OKRs :).